akka-stream-tests/src/test/scala/akka/stream/scaladsl/FramingSpec.scala

```diff
@@ -5,16 +5,15 @@ package akka.stream.scaladsl
 
 import java.nio.ByteOrder
 import java.util.concurrent.ThreadLocalRandom
-import java.util.concurrent.atomic.AtomicLong
 
+import akka.stream._
 import akka.stream.scaladsl.Framing.FramingException
 import akka.stream.stage.{ GraphStage, _ }
 import akka.stream.testkit.{ StreamSpec, TestPublisher, TestSubscriber }
-import akka.stream._
 import akka.util.{ ByteString, ByteStringBuilder }
+import org.scalatest.concurrent.PatienceConfiguration.Timeout
```

Comment:
`implicit val patience = PatienceConfig(testKitSettings.DefaultTimeout.duration)` in `AkkaSpec`